nightfall-client/src/repositories/commitment-repository.mjs

```diff
@@ -0,0 +1,48 @@
+import config from 'config';
```

Comment:
@daveroga Yes the idea is refactor:
Service folder will contain our controllers to do some logic (filter, verification ... )
And Repository is to have only communication with the database. 

At the beginning it will stay a little bit confuse. But in the final of the day the idea is have all the code refactored for the new structure.

Reply:
Ok

Reply:
@daveroga In my vision, really, we should start to break the responsibilities at least for the new code. And we can do the refactor progressively. For me is easier keep how as is now. But I really prefer to start the refactoring progressively.